apps/saleor-app-checkout/utils/debug.ts

```diff
@@ -0,0 +1,3 @@
+import debugPkg from "debug";
+
+export const createDebug = (namespace: string) => debugPkg.debug(`saleor-app-checkout:${namespace}`);
```

Comment:
Do we really want to use that library? If we want to use something more robust, then maybe we should consider instead:
- https://github.com/pinojs/pino
- https://github.com/winstonjs/winston

They have features like redaction of sensitive data, transport (to save logs in external source - e.g. DataDog), streaming logs in real time

One more thing: personally, I'm not a fan of `%` formatting in error messages - JS already has a built-in feature for that, template strings :P

Reply:
I agree and this is out of scope of this PR.